getting_hired/preparing_for_job_search/professional_networking.md

```diff
@@ -0,0 +1,82 @@
+
+
+Think of it from the company’s perspective. Hiring is a pain. In a perfect world, a great candidate would show up instantly any time a new job opened up. This would save them going through the process of posting the job, screening applicants, marking technical assessments and conducting interviews. This all takes time and is very expensive. So it stands to reason why most companies would prefer to hire through the hidden jobs market. 
+
+So, how do you tap into the hidden jobs market? Through your network. If your network sees you as a trusted individual with a particular skill set, you can be sure they will think of you when matching opportunities arise. You might ask around about potential jobs and receive a heads up of a new opportunity at someone’s company. Better still, you could be recommended for a particular role. Many companies give bonuses to staff who refer a candidate that gets hired. So, the incentive to network and pass names along goes both ways.
+
+This is not all to say you should refrain entirely from applying on posted jobs. These can still be worthwhile to pursue, but we recommend quality over quantity and seeking companies that you find interesting and have core values you identify with. You will be better off sending fewer, tailored applications over hundreds of the same application in a shotgun-like approach. We cover more on this in the Getting Hired course.
+
+### How to build a professional network
+
+Just like building your technical skills, networking requires careful attention and a strategic approach. A local network allows you to build strong, in-person connections while a virtual network allows you to connect with people all over the world. Both are useful and a strategy that focuses on the two will be the most robust.
+
+#### Start Now
+Your network will take time to build. The sooner you start to make connections, the better. Starting now means you will have more of a network to leverage when you need to rely on it. Consider proactively allocating a percentage of your web development study time to this activity. This can act as a nice breather between the mental workout of coding, and you may even combine networking and learning during activities like attending local meetups and conferences.
+
+#### Identify Your Options
+Your time is limited and your options are endless. So, you should understand what options you have at your disposal and select one or two to start with. 
+
+In-person networking opportunities will vary based on your location. Larger towns and cities will likely have well-established tech-related events and groups, whereas smaller communities may be more limited. To get a better sense of what’s going on around you, consider the following:
+
+* Check [MeetUp](https://www.meetup.com/) and/or Google for communities focused on a given technology.
```

Comment:
Suggest adding LinkedIn as a good option to find local events/groups.

Reply:
done